kong/pdk/request.lua

```diff
@@ -12,6 +12,7 @@ local phase_checker = require "kong.pdk.private.phases"
 
 
 local ngx = ngx
+local var = ngx.var
 local sub = string.sub
 local find = string.find
 local lower = string.lower
```

Comment:
Now we don't use `lower`, just remove it for luacheck.

Reply:
It's not here that `lower` is added. There're other uses of lower in the code.